test/ecto/changeset_test.exs

```diff
@@ -1061,6 +1061,26 @@ defmodule Ecto.ChangesetTest do
     assert changeset.errors == [topics: {"can't be blank", [validation: :required]}]
   end
 
+  test "filter_required/2" do
+    required_fields =
+      changeset(%{"body" => "something"})
+      |> filter_required([:title, :body])
+
+    assert [:body] == required_fields
```

Comment:
nitpick: either `assert actual == expected` or `assert expected = actual` but not `assert expected == actual`, wdyt?

```suggestion
    assert required == [:body]
```